
################################################################################################################ 
########### LED MANAGER NODE
################################################################################################################

led_manager:

  white: 0 
  blue: 5 
  yellow: 5 
  pink: 4 
  green: 3 
  orange: 6 
  red: 2
  black: -1

  waypoint_goal: 0  # Goals points that the robot must signal
  ghost_goal: 1 


################################################################################################################ 
########### SAFE TWIST
################################################################################################################

safe_twist: 

  safe_distance: 100    # distance in cm 
 
  motor_brake_factor: -1 
 
  max_linear_speed: 2.0  
  max_angular_speed: 20.0

  disable_ultrasonics: false



################################################################################################################ 
########### JOY ESP INTERFACE
################################################################################################################

joy_esp_interface: 

  max_speed_joy_linear: 5 
  max_speed_joy_angular: 10 
  
  max_value_controller: 127 
  
  drift_analog_tolerance: 20 




################################################################################################################ 
########### ODOMETRY
################################################################################################################

odometry:
  wheels_track: 0.3800  # distance in meters 
  wheels_radius: 0.075  # distance in meters 
  ticks_per_revolution: 7200 # 2400 * 3
  base_link_offset: 0.08 # distance (in meters) between the base_footprint and base_link 


  
